manifests/config.pp

```diff
@@ -157,66 +157,61 @@
     # the vast majority of files here are init-files
     # so any change there should trigger a full service restart
     if $prometheus::server::restart_on_change {
-      File {
-        notify => Class['prometheus::run_service'],
-      }
-      $systemd_notify = [Exec['prometheus-systemd-reload'], Class['prometheus::run_service']]
+      $notify = Class['prometheus::run_service']
```

Comment:
we should now use a selector statement here?

Reply:
https://github.com/alexjfisher/puppet-prometheus/pull/1

Reply:
sure.  Updated.  I could always go either way when deciding if a selector is more readable than a simple `if/else` when the control expression is a Boolean.